sources/packages/web/src/types/contracts/StudentContract.ts

```diff
@@ -13,6 +14,7 @@ export type StudentProfile =
       | InstitutionStudentProfileAPIOutDTO
       | AESTStudentProfileAPIOutDTO
     ) & {
+      legacyProfile?: LegacyStudentProfileAPIOutDTO;
```

Comment:
`AESTStudentProfileAPIOutDTO` has the legacyProfile in it already. 

The problem is using `|` which creates the intersection of the types. Instead using `&` would create the union of the types.

```ts
export type StudentProfile = (StudentProfileAPIOutDTO &
  InstitutionStudentProfileAPIOutDTO &
  AESTStudentProfileAPIOutDTO) & {
  birthDateFormatted: string;
};
```

Reply:
The goal is to use the intersection otherwise, a mandatory property from one type would become mandatory to `StudentProfile`, which would not represent the accurate object.
The `StudentProfile` represents what is common between the three types, plus what can optionally be available and optionally can be consumed by the UI.